pyproject.toml

```diff
@@ -31,6 +31,7 @@ tqdm = "^4.62.3"
 xarray = "^0.20.2"
 joblib = "^1.2.0"
 jupyter = {version = "^1.0.0", extras = ["notebook"], optional = true}
+jupyterlab = {version = ">=3.2.9, <4.0.0", extras = ["notebook"], optional = true}
```

Comment:
do newer versions of jupyter lab give problems? What is incompatible>

Reply:
In versions above 4.0, for some reason widgets seem to not properly output anymore. I don't have any idea why, and websearches don't give recent hits. Even graphs with "%matplotlib widget" don't show, but give something like this:
![grafik](https://github.com/OpenCOMPES/sed/assets/53396064/efe0e440-8c02-4b90-bc92-dcf46cff6a04)
Feel free to test yourself, the pinning is a rather dirty fix...

Reply:
Make sure that the versions of 'ipympl', 'matplotlib' and 'ipywidgets' from Jupyter and the one you use in your environment do match. That should fix the problem

Reply:
Thanks, indeed the limit on ipywidget was te problem. I removed this, now it works.